### ISSUE TITLE
Add additional fields to review question

### ISSUE THREAD

**Issue by martzcodes** [martzcodes/OMSCentral#44]
- Which languages does a course use
- What number of course was this for the reviewer (i.e. first course, etc)
- Recommend for first class
- Other courses taken at same time?

**Comment by peridot121** [comment]
I would like to see some way of tracking how the course changed. You see reviews of CN and AI4R which say it's a breeze and they spend 5 hours every other week on it. Then they change the course, and it surprises people. This could be an additional field of course revamped from last semester, or a graph of course difficulty over time.

**Comment by martzcodes** [comment]
@peridot121 good idea... could add a button to "report revamp" or something and if X% of reviewers choose that in a semester it flags the course.

At some point I'd also like to make a few graphs regarding difficulty / etc by semester

**Comment by carmineguida** [comment]
- Does the class use Proctortrack 
- Does the class have group projects
- Does the class require purchasing anything (books, software, etc.)

**Comment by michaelrswenson** [comment]
Are the tests open book / notes / internet? I agree with all other recommendations made as well, especially tracking language(s) used, use of ProctorTrack, whether any/all projects are group.

**Comment by michaelrswenson** [comment]
Oh yes: Number of projects / assignments / homeworks, whether there are quizzes (and how many).

**Comment by waciumawanjohi** [comment]
It would be useful to know the due dates of the assignments/dates of tests, to facilitate a semester planner that helps students avoid courses that hold tests in the same week.

**Comment by martzcodes** [comment]
@waciumawanjohi I think due dates would be pushing it... that should really be up to the syllabus / course and would be hard to moderate

**Comment by set-list** [comment]
Professor for a course for a specific semester would also be helpful. Courses can change depending on the professor's teaching style
